SF-Managed-Standard-SKU-1-NT-MI/azuredeploy.json

```diff
@@ -66,8 +66,11 @@
             "type": "String"
         },
         "vmIdentityRoleNameGuid": {
-            "defaultValue": "[newGuid()]",
-            "type": "String"
+            "defaultValue": "0000000-0000-0000-0000-000000000000",
```

Comment:
Do you think it's better to not have default value and force the user to create a new one on their own? You can also mention this in the description

Reply:
yeah, I removed the default value. And on the readme I noted that it should be given with a guild format